### PR TITLE
added-portrait-file

### PR DESCRIPTION
**Pull request by murtazshaikh**
review pr...

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Text Typography</title>
+</head>
+<body>
+    <div class="container">
+        <img src="img/face.jpg" alt="face">
```
Comment (nehalskdev) [inline index.html:12]: add ``` html 
<alt>```

**Review by nehalskdev — APPROVED** [review]
good one

**Review by OSAMA478 — APPROVED** [review]
Nice...

**Review by mmusk — APPROVED** [review]
LGTM

**Comment by syedabdullahali** [comment]
nice one

**Pull request merged**